orders/OrderFormNotion.ts

```diff
@@ -68,4 +78,14 @@ export default class OrderForm {
             console.warn(`Order form checker: An error occurred when checking for order form updates.`, e);
         }
     }
+
+    private completedSushiOrder(orderObject: PageObjectResponse) {
+        if (orderObject.properties["Order Description"]?.type == "title" && orderObject.properties["Product Name"]?.type == "rich_text" && orderObject.properties["Subtotal"]?.type == "number") {
+            const title = orderObject.properties["Order Description"]?.title.at(0)?.plain_text ?? null;
+            const subtotal = orderObject.properties["Subtotal"].number ?? null;
+
+            return title && title != "<Seller / Product Description> NOT your name" && subtotal;
```

Comment:
If TypeScript is happy to booleanify this then it's fine but maybe we can make it explicitly boolean:

```suggestion
            return title != "<Seller / Product Description> NOT your name" && subtotal != null;
```